### PR TITLE
add setup for titiler metapackage

### LINKED ISSUE #294: Publish Titiler as metapackage
We now have 3 `namespaces packages`: `titiler.core`, `titiler.mosaic`, `titiler.application` published on pypi with version 0.3.0

Having both `old` titiler==0.2.0 and the namespaces package with 0.3.0 is really confusing. As @kylebarron suggested we could publish a `empty` metapackage with the same version as the sub package.

### PR DESCRIPTION
**Pull request by vincentsarago**
closes #294

### CODE REVIEW

File: scripts/publish [hunk 1]
```diff
@@ -15,3 +15,8 @@ do
     twine upload dist/*
     popd
 done
+
+# Publish Titiler meta package
+rm -rf dist
+python setup.py sdist
+twine upload dist/*
```
Comment (kylebarron) [inline scripts/publish:22]: Note you could also create a `titiler/metapackage` folder to have the metapackage "as a module" instead of at the top level.

**Review by kylebarron — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        "Programming Language :: Python :: 3.9",
+    ],
+    keywords="COG STAC MosaicJSON FastAPI Tile Server Dynamic",
+    author="Vincent Sarago",
+    author_email="vincent@developmentseed.org",
+    url="https://github.com/developmentseed/titiler",
+    license="MIT",
+    zip_safe=False,
+    install_requires=inst_reqs,
+    packages=[],
```
[the remark this answers is not in the document]
Comment (vincentsarago) [reply setup.py:40]: they use distutils, not sure if it's different 🤷

File: scripts/publish (hunk 1 above)
Comment (vincentsarago) [reply scripts/publish:22]: yeah, might be confusing 🤷

File: setup.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kylebarron) [reply setup.py:40]: Ah I see. I think `packages=[]` is probably best then.

**Pull request merged**